Fix documentation typo

bzier opened the pull request:
`PromethesuRegistry` -> `PrometheusRegistry`

bzier commented:
I expect this is a simple thing to get merged, but please let me know if you need anything from me. I see a message that "merge is not an allowed merge method in this repository". If I need to reopen the PR with different options, I'm happy to do so.

cc: @fstab @dhoard @tomwilkie @zeitlinger

zeitlinger commented:
> I expect this is a simple thing to get merged, but please let me know if you need anything from me. I see a message that "merge is not an allowed merge method in this repository".

not sure why you see this - just accept the licence button, then it should be merged.

bzier commented:
@zeitlinger Thanks, I've addressed the DCO issue now. The guide suggested a rebase to do the sign-off. The push reset your auto-merge 🤷  I also updated the branch with the latest commits from over night, so it is current as I write this.

Here is what I see at the moment:
![image](https://github.com/user-attachments/assets/d49c89a9-cd93-4ab7-9d2f-c4b5897bdc05)

Let me know if there's anything else I should do. Thanks.
